tsconfig.json

```diff
@@ -8,7 +8,7 @@
         "outDir": "built",
         "sourceMap": true,
         "inlineSources": true,
-        "strict": true,
+        "strict": false,
```

Comment:
why's this needed?

Reply:
turning off the typings so it doesn't yell at all of the any's......happy to take suggestions of a better way